ScreenSleeper/Program.cs

```diff
@@ -21,6 +21,9 @@ public class Program
 {
 	static void Main(string[] args)
 	{
+		if (Application.IsRunning)
+			return;
```

Comment:
Super small, nitpicky formatting request: could you please put even single-line code blocks inside braces? 🙂 Thank you so much!